scala/friesian/scripts/setup_redis_master_replicas.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+master_server='172.168.3.116'
```

Comment:
It's better reading these IPs from parameters and providing a simple example to user.